src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,41 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+
+import duke.Parser;
+
+/**
+ * Represents the subclass of Task. It contains the description of the task and the date of the
+ * event.
+ */
+public class Event extends Task {
+    private final LocalDate eventTime;
+
+    /**
+     * Constructs a Event class that contains the description of the task and the date of the event.
+     * @param description description of the Event.
+     * @param eventTime date when the Event occurs.
+     */
+    public Event(String description, LocalDate eventTime) {
+        super(description);
+        this.eventTime = eventTime;
+    }
+
+    /**
+     * Returns data representation of the Event to be saved into the save file.
+     * @return data representation of the Event.
+     */
+    @Override
+    public String data() {
```

Comment:
Perhaps it would be more comprehensive if this method name starts with a verb. 
I noticed the same issue in several other places too.